src/intl/messages.js

```diff
@@ -566,10 +584,15 @@ export const messages: { [messageId: string]: MessageType } = {
   selectTheBootableDeviceTooltip: 'Select the bootable devices in the order you would like them to boot.',
   sendShortcutKey: 'Send Key',
   sendCtrlAltDel: 'Ctrl+Alt+Del',
+  serialConsole: 'Serial Console',
+  serialConsoleOptions: 'Serial Console Options',
```

Comment:
Serial console settings is not needed on web-ui

Reply:
AFAIK SSH key is for serial console. The original label was a bit more descriptive - now it's reflected in the tooltip only. Originally it was planned in Console section.

Reply:
I prefer the current design (which fits the current implementation).
It can be in the console section.

Reply:
We might end up with leaving the current design but I would like to have some plan B here. Can we include them for now and decide later?